samples/bot-daily-task-reminder/csharp/README.md

```diff
@@ -0,0 +1,112 @@
+﻿﻿﻿
+## Bot daily task reminder
+
+This sample shows feature where user can schedule a recurring task and get reminder on the scheduled time.
+
+![Schedule task ](BotDailyTaskReminder/Images/ScheduleTaskCard.png)
```

Comment:
Add the header for this image and also mention the command name in doc for "create-reminder"